src/sirhelpers.c

```diff
@@ -341,7 +341,11 @@ bool _sir_getchar(char* input) {
         return false;
 
 #if defined(__WIN__)
+# if defined(__EMBARCADEROC__)
```

Comment:
What's up with this? It can't call _getch?

Reply:
Nope, only getch()!